extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -96,28 +96,70 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
         });
     }
 
+    _isBucketdEntry(entry) {
+        // raft log entries
+        // {
+        //   bucket: bucket name
+        //   key: no key value
+        //   value: bucket value
+        // }
+        return entry.key === undefined && entry.bucket !== METASTORE;
+    }
+
+    _isMongoEntry(entry) {
```

Comment:
Mongo entry? Is this for 7.x branches?

Reply:
Yes, should not be needed normally as it's 8.x specific (and should already be compatible)

Reply:
`MongoEntry` may be a confusing name as `7.x` won't be supporting mongo. I can rename to `FileMDEntry`.